api/application.py

```diff
@@ -49,6 +52,39 @@ def sesam_run():
 
     return jsonify(output_payload)
 
+@application.route('/api/matlab', methods=['POST'])
+def matlab_test():
+    if not request.json:
+        abort(400)
+
+    with NamedTemporaryFile(delete=False, suffix='.json', mode='wt') as inf:
```

Comment:
I was confused with `inf` here at first glance, thinking it was mathematical infinity. I'd rename to `infile` or similar just to remove that initial confusion.

Reply:
oh you mathematicians :)